client/src/interface/SchedulerProps.ts

```diff
@@ -0,0 +1,50 @@
+import { Moment } from 'moment-timezone';
+
+type changeEventHandler = (args: React.ChangeEvent<{ value: unknown }>) => void;
+type clickEventHandler = (args: React.MouseEvent<HTMLButtonElement>) => void;
+type strBoolFunc = (args: string) => boolean;
+type simpleFunc = () => void;
+
+export interface disableDateProp {
+  activeStartDate: Date;
+  date: Date;
+  view: string;
+}
+
+export interface schedUrlProp {
+  username: string;
+  duration: string;
+}
+
+export interface completionUrlProp {
+  appointID: string;
+}
+
+export interface confirmProp {
+  username: string;
+  duration: number;
+  timeZone: string;
+  time: Moment;
+  // checkConfirmation: clickEventHandler;
```

Comment:
Is this not needed anymore?

Reply:
Yes, it is not needed anymore. I should have removed it but I missed it.